README.md

```diff
@@ -17,7 +17,7 @@ tools for scientific python:
 
 ```
 conda install jupyter numpy matplotlib --yes
-conda install -c lightsource2 lmfit --yes
+conda install -c conda-forge lmfit
```

Comment:
Combine lines 19 and 20 so that it says

`conda install -c conda-forge lmfit jupyter numpy matplotlib --yes`

Thanks!